src/System.IO.FileSystem/tests/File/GetSetAttributes.cs

```diff
@@ -101,5 +101,40 @@ public void WindowsInvalidAttributes(FileAttributes attr)
             Set(path, attr);
             Assert.Equal(FileAttributes.Normal, Get(path));
         }
+
+        [SkippableFact]
```

Comment:
We're not using this anywhere else in the repo.  We do have something called [ConditionalFact(..)] which lets you give it a method that'll be run to determine whether to execute the test, and I'd prefer we use that here.  Seems like you can just create a helper like:
```C#
private static bool CanCreateSymbolicLinks { get { ... } }
```
and then use it like:
```C#
[ConditionalFact("CanCreateSymbolicLinks")]
public void SymLinksAreReparsePoints()
{
}
```
etc.